docs/namespaces.md

```diff
@@ -74,4 +74,5 @@ will all use the default namespace, which is simply named "`default`".
 ```
 
 You can also use the `CONTAINERD_NAMESPACE` environment variable to specify the default namespace to use for
-any of the `ctr` client commands.
+any of the `ctr` client commands.  
+>To know more about Namespaces package click here [`Namespaces`](https://pkg.go.dev/github.com/containerd/containerd/v2/pkg/namespaces).
```

Comment:
link already provided above.. 
```suggestion
```
You could link to some k8s.io namespace stuff .. also there is moby and probably some other changes need to be made.  Cheers.

Reply:
>To know more about different Namespaces packages click below links :  
>[`Containerd Namespaces Pkg`](https://pkg.go.dev/github.com/containerd/containerd/v2/pkg/namespaces).  
>[`kubernetes Namespaces Pkg`](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/).  
>[`Moby(Docker) Namespaces Pkg`](https://docs.docker.com/engine/security/userns-remap/).

I have updated the other links too in a diff manner please re-verify